aptos-move/aptos-transactional-test-harness/src/aptos_test_harness.rs

```diff
@@ -471,7 +471,8 @@ impl<'a> AptosTestAdapter<'a> {
     /// Should error if the transaction ends up being discarded, or having a status other than
     /// EXECUTED.
     fn run_transaction(&mut self, txn: Transaction) -> Result<TransactionOutput> {
-        let mut outputs = AptosVM::execute_block_and_keep_vm_status(vec![txn], &self.storage)?;
+        let mut outputs =
+            AptosVM::execute_block_and_keep_vm_status(vec![txn], &self.storage, true)?;
```

Comment:
@wrwg, this new flag set to `true` indicates that the VM should include #[test_only] native functions. I'm assuming this is needed for your `aptos-transactional-test-harness`.